models/sipnet/R/model2netcdf.SIPNET.R

```diff
@@ -32,11 +32,18 @@ model2netcdf.SIPNET <- function(outdir, sitelat, sitelon, start_date, end_date,
   ### Determine number of years and output timestep
   start.day <- sipnet.output$day[1]
   num.years <- length(unique(sipnet.output$year))
+  # quick consistency check
+  if (length(lubridate::year(end_date):lubridate::year(start_date))!=num.years) {
+    PEcAn.logger::logger.severe("Date range specified in model2netcdf.SIPNET() function call and total number of years in SIPNET output are not equal")
+  }
   years <- unique(sipnet.output$year)
-  out.day <- length(which(sipnet.output$year == years[1] & sipnet.output$day == start.day))
+  # check that specified years and output years match
+  if (!all(years %in% lubridate::year(start_date):lubridate::year(end_date))) {
```

Comment:
DRY. Year sequence is used twice -- I would make it its own variable.

```
year_seq <- seq(lubridate::year(start_date), lubridate::year(end_date))
if (length(year_seq) != num.years) {...}
if (!all(years %in% year_seq)) {...}
```

Reply:
Good catch, will check that out